addons/product/views/product_views.xml

```diff
@@ -58,13 +58,13 @@
                         <label for="name" string="Product"/>
                         <h1>
                             <div class="d-flex">
-                                <field name="is_favorite" widget="boolean_favorite" class="me-3" nolabel="1"/>
+                                <field name="is_favorite" widget="boolean_favorite" class="me-2" nolabel="1"/>
                                 <field class="text-break" name="name" options="{'line_breaks': False}" widget="text" placeholder="e.g. Cheese Burger"/>
                             </div>
                         </h1>
                     </div>
                     <div name="options">
-                        <span class="d-inline-flex">
+                        <span name ="sale_option" class="d-inline-flex">
```

Comment:
:eyes:

Reply:
Ouiii? :innocent: Just to respect the order between the checkboxes on the product page and filters. 
![image](https://github.com/user-attachments/assets/81e4b20d-a513-4e23-a8c7-2d5314870b5b)
![image](https://github.com/user-attachments/assets/511ee62d-ed98-4480-81ea-d7ad016fba6a)